projects/observability/src/shared/components/explore-query-editor/explore-visualization-builder.ts

```diff
@@ -130,6 +138,7 @@ export class ExploreVisualizationBuilder implements OnDestroy {
       interval: this.resolveInterval(state.interval),
       filters: state.filters && this.graphQlFilterBuilderService.buildGraphQlFieldFilters(state.filters),
       groupBy: state.groupBy,
+      orderBy: state.orderBy && [{ direction: state.orderBy, key: state.series[0].specification }],
```

Comment:
Can we use optional chaining here? Not sure if the values will be undefined

Reply:
Are you referring to the `series` part here?

It refers to the first metric in the visualisation. That will always be defined since we always have at least one metric.

<img width="480" alt="Screenshot 2022-07-12 at 12 05 37 PM" src="https://user-images.githubusercontent.com/29686866/178424682-ccf9e1e8-0910-49c9-be0f-a2a5fcd71364.png">

Let me add an optional chaining anyway, doesn't hurt.

Reply:
Done ✅